namespaces/live.cloud-platform.service.justice.gov.uk/courtfines-prod/resources/route53direct.tf

```diff
@@ -0,0 +1,101 @@
+resource "aws_route53_zone" "route53_direct_zone" {
+  name = "courtfines.direct.gov.uk"
+
+  tags = {
+    business-unit          = var.business_unit
+    application            = var.application
+    is-production          = var.is_production
+    environment-name       = var.environment
+    owner                  = var.team_name
+    infrastructure-support = var.infrastructure_support
+    namespace              = var.namespace
+  }
+}
+
+resource "kubernetes_secret" "route53_direct_zone_sec" {
+  metadata {
+    name      = "route53-direct-zone-output"
+    namespace = var.namespace
+  }
+
+  data = {
+    zone_id = aws_route53_zone.route53_direct_zone.zone_id
+  }
+}
+
+resource "aws_route53_record" "aws_route53_record_prod" {
+  name    = "courtfines.direct.gov.uk."
+  zone_id = aws_route53_zone.route53_direct_zone.zone_id
+  type    = "A"
+  alias {
+    zone_id                = "ZHURV8PSTC4K8"
+    name                   = "dualstack.court-loadb-8mcola2l2by0-173012739.eu-west-2.elb.amazonaws.com."
+    evaluate_target_health = true
+  }
+}
+
+resource "aws_route53_record" "aws_route53_record_ns" {
```

Comment:
The NS record is not needed. The hosted_zone creation generates the NS record which is passed to the root domain

Reply:
Removed.